build.assets/version-check-prerelease/main.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Could you convert this to a proper godoc comment? Something like:

```
// Command version-check-prerelease exits non-zero when given a 
// git tag that is a prerelease. This allows us to avoid publishing
// releases for internal builds.
```

Reply:
Thanks! Fixed in 6b3d86a.